src/entity/Virus.js

```diff
@@ -32,6 +32,7 @@ Virus.prototype.onEat = function (prey) {
 };
 
 Virus.prototype.onEaten = function (c) {
+    if (!c.owner) return; // Only players can explode

```

Comment:
Prevents crash when mothercells eat viruses, because mothercells are not supposed to explode like players do.